backend/drive/drive.go

```diff
@@ -710,30 +722,31 @@ func (f *Fs) List(dir string) (entries fs.DirEntries, err error) {
 				return true
 			}
 			entries = append(entries, o)
-		case isDocument:
+		case *driveSkipGdocs:
+			fs.Debugf(remote, "Skipping google document")
```

Comment:
When `*driveSkipGdocs` is set, all 'unknown' files will be caught by this case.
This should probably log the mime-type or use the same format as line 730.

Reply:
Thanks - will do :-)